lib/cybersource/api.rb

```diff
@@ -276,6 +279,20 @@ def should_credit?(kb_payment_id, context, options = {})
         (now - transaction.created_at) >= threshold
       end
 
+      def before_gateways(kb_transaction, last_transaction, payment_source, amount_in_cents, currency, options, context = nil)
+        # Provide necessary information for merchant descriptor
+        if options[:merchant_descriptor].present? && options[:merchant_descriptor]['is_amex'].nil?
```

Comment:
My [remark](https://github.com/killbill/killbill-cybersource-plugin/pull/7#discussion_r70713412) still holds:

* Check the plugin property `cc_type` instead of making the user specify another property inside the descriptor
* Check if `respond_to?(:brand)` instead below, for performance reasons (the vast majority of calls would throw an unnecessary exception otherwise)